docs/version-compatibility.md

```diff
@@ -12,7 +12,7 @@ Supports XR 7.7.1 and 7.8.1 (the first and most recent released versions of XRd)
 
 - Does not support the `--boot-log-level` arg in `launch-xrd`.
 - Requires the host cgroup mount to be passed into the container manually:
-  - Use '`--args '-v /sys/fs/cgroup:/sys/fs/cgroup:ro'`' to `launch-xrd`.
+  - Pass the docker/podman arguments `-v /sys/fs/cgroup:/sys/fs/cgroup:ro` through `launch-xrd`.
```

Comment:
Will this be passed through? `launch-xrd` has it's own `-v` which will pick it up instead.

Reply:
Oh no :'(

Reply:
Yep, I was going to suggest that